src/components/UserCard/UserCard.css

```diff
@@ -64,16 +65,52 @@
 
 .links {
   @apply --marketplaceH4FontStyles;
+  margin-top: 13px;
+
+  @media (--viewportMedium) {
+    margin: 16px 0 0 0;
+  }
 }
 
 .withBioMissingAbove {
   @media (--viewportMedium) {
     /* Fix baseline alignment if bio is missing from above */
-    margin-top: 18px;
+    margin-top: 16px;
   }
 }
 
 .linkSeparator {
   margin: 0 10px;
   color: var(--marketplaceColor);
 }
+
+.headingRow {
+  display: flex;
+  flex-direction: column;
+
+  @media (--viewportMedium) {
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: baseline;
+  }
+}
```

Comment:
Could you check that the row is always divisible by 6px (and 8px on the desktop layout)? If not, add some padding to top and bottom so that it's size will match that requirement and texts are baselined nicely.

We should be more pedantic with our baseline rules so that there's less need for random margins between elements (i.e. `margin-top: 12px` makes sense, but `13px` is wrong -> some other component has wrong height).

Reply:
@Gnito The component is now internally in baseline. However, I'll see if the css definitions could be improved a bit more to have more logical values regarding the baseline definitions.